crown-court-proceeding/src/main/java/uk/gov/justice/laa/crime/crowncourt/service/MaatCourtDataService.java

```diff
@@ -250,4 +251,15 @@ public RepOrderCCOutcomeDTO createOutcome(RepOrderCCOutcomeDTO outcomeDTO, Strin
         log.info(String.format(RESPONSE_STRING, response));
         return response;
     }
+
+    public Long outcomeCount(Integer repId, String laaTransactionId) {
```

Comment:
Is there a special reason for the return type to be `Long` instead of `long`? It is preferable to be `long` as it is guaranteed to be non-null and reduces the need for null checks elsewhere in the code base